packages/graphics/opengl-mali/package.mk

```diff
@@ -17,13 +17,21 @@
 ################################################################################
 
 PKG_NAME="opengl-mali"
-PKG_VERSION="20130520"
+PKG_VERSION="4412_r5p0"
+PKG_DEPENDS_TARGET="toolchain"
+if [ "$DISPLAYSERVER" = "x11" ]; then
+  PKG_VERSION="${PKG_VERSION}_x11"
+  PKG_URL="http://people.piment-noir.org/~fraggle/download/odroid/$PKG_NAME-$PKG_VERSION.tar.xz"
+  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET libXfixes libXrender libXcomposite libxcb"
+else
+  PKG_VERSION="${PKG_VERSION}_fbdev"
+  PKG_URL="http://people.piment-noir.org/~fraggle/download/odroid/$PKG_NAME-$PKG_VERSION.tar.xz"
+fi
+PKG_SOURCE_DIR="${PKG_NAME}-${PKG_VERSION}"
```

Comment:
When this was written, I was constantly switching between fbdev and x11 flavors during testing.  If you plan to pick finalize your pick with _x11, then you can drop the _fbdev lines completely.